test/bats/azure.bats

```diff
@@ -64,13 +61,14 @@ setup() {
 }
 
 @test "install driver helm chart" {
-  run helm install csi-secrets-store ${SECRETS_STORE_CSI_DRIVER_PATH}/charts/secrets-store-csi-driver --namespace dev --set windows.enabled=true
-  assert_success
-}
+  run helm install csi charts/csi-secrets-store-provider-azure --namespace dev --set windows.enabled=true \
+      --set secrets-store-csi-driver.windows.enabled=true \
+      --set image.repository=${PROVIDER_TEST_IMAGE} \
+      --set image.tag=${IMAGE_TAG} \
+      --set image.pullPolicy="IfNotPresent" \
+      --dependency-update
 
-@test "install azure provider with e2e image" {
-  yq w -d 1 ${PROVIDER_YAML} "spec.template.spec.containers[0].image" "${PROVIDER_TEST_IMAGE}:${IMAGE_TAG}" \
```

Comment:
why do we want this removed?

Reply:
The driver and provider are installed using the helm charts, so we no longer need to install using `kubectl apply -f`

Reply:
Issue to track testing with deployment yamls - https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/196

Reply:
This issue is being tracked [here](https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/196) to unblock this PR.